src/config.rs

```diff
@@ -101,7 +100,6 @@ impl Default for StarknetGeneralConfig {
             default_eth_price_in_fri: 1_000_000_000_000_000_000_000,
             constant_gas_price: false,
             sequencer_address: contract_address!(SEQUENCER_ADDR_0_13_0),
-            cairo_resource_fee_weights: Arc::new(HashMap::from([(N_STEPS_RESOURCE.to_string(), 1.0)])),
```

Comment:
This was removed in `cairo-lang`: https://github.com/starkware-libs/cairo-lang/commit/efa9648f57568aad8f8a13fbf027d2de7c63c2c0